shared/tsconfig.base.json

```diff
@@ -1,9 +1,8 @@
 {
   "compilerOptions": {
-    "outDir": "${configDir}/build",
-    "noEmit": true,
```

Comment:
All projects must emit the results of `tsc` to use project references. The files are emitted to this `node_modules/.cache/.tsc_cache` folder, so you won't notice it typically during development. Most of our projects were already doing this.